assets/src/components/map/follower.tsx

```diff
@@ -181,7 +195,7 @@ export const usePickerContainerFollowerFn = () => {
 
 export const drawerOffsetAutoCenter =
   (useCurrentZoom: boolean): UpdateMapFromPointsFn =>
-  (map, points) => {
+  (map, points, shouldOffset) => {
```

Comment:
> https://github.com/mbta/skate/pull/2286#discussion_r1399475866

Maybe something like this? 

TBH, I'm not even sold that points should be an argument to this function or the base <Follower/> but I think that's a slightly larger refactor.
```suggestion
  (useCurrentZoom: boolean, topLeft: Point): UpdateMapFromPointsFn =>
  (map, points) => {
```

And then remove the [`topLeft` definition](https://github.com/mbta/skate/pull/2286/files#diff-fec870c97fb7c88e5929bb4949758d5dbbc8137119df5b14a1efcfe744971ab0R217)